roles/web-server.json

```diff
@@ -4,25 +4,30 @@
   },
   "json_class": "Chef::Role",
   "run_list": [
+    "recipe[vim]",
     "recipe[git]",
-    "recipe[apt]",
+    "recipe[yum]",
+    "recipe[yum-epel]",
+    "recipe[selinux::permissive]",
     "recipe[apache2]",
-    "recipe[apache2::mod_php5]",
     "recipe[apache2::mod_env]",
     "recipe[apache2::mod_deflate]",
     "recipe[apache2::mod_expires]",
     "recipe[apache2::mod_rewrite]",
     "recipe[apache2::mod_status]",
+    "recipe[apache2::mod_access_compat]",
+    "recipe[apache2::mod_proxy]",
+    "recipe[apache2::mod_proxy_fcgi]",
     "recipe[php]",
     "recipe[php::module_gd]",
     "recipe[php::module_ldap]",
     "recipe[php::module_curl]",
     "recipe[php::module_mysql]",
-    "recipe[phpredis]",
+    "recipe[php-fpm]",
+    "recipe[yum-mysql-community::mysql55]",
```

Comment:
Yeah - I agree - the community MySQL cookbook is written to really want to install vanilla mysql for whatever reason, so that's what I did. I think we can have that done by Sunday.